src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java

```diff
@@ -254,6 +254,11 @@ else if (participantType == ParticipantType.edge)
                 ops.addArguments("window-size=1200x600");
             }
 
+            if (options.getDisableDecoding())
+            {
+                ops.addArguments("force-fieldtrials=WebRTC-BlackFrameDecoder/Enabled/");
```

Comment:
beware, the current plan is to enable this as a separate flag for consistency with other decoders:
`--use-black-frame-decoder-for-peer-connection`
-- will update the webrtc CL accordingly